src/Material.h

```diff
@@ -16,6 +16,9 @@
 #include "log.h"
 #endif
 
+/** The minimum allowable total cross-section value for any group */
```

Comment:
Sorry to be nit picky, but could you change this comment in `Material.h` as well?